corehq/apps/userreports/expressions/utils.py

```diff
@@ -9,6 +11,11 @@ def safe_pow_fn(a, b):
 SAFE_OPERATORS = copy.copy(DEFAULT_OPERATORS)
 SAFE_OPERATORS[ast.Pow] = safe_pow_fn  # don't allow power operations
 
+FUNCTIONS = DEFAULT_FUNCTIONS
+FUNCTIONS.update({
+    'timedelta_to_seconds': lambda x: x.total_seconds() if isinstance(x, timedelta) else None
+})
```

Comment:
I did, seems logical to me. was going to suggest updating the docs in case others want to do similar stuff